src/Sylius/Bundle/ChannelBundle/Resources/config/services.xml

```diff
@@ -17,10 +17,21 @@
                                http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <parameters>
+        <parameter key="sylius.form.type.channel.class">Sylius\Bundle\ChannelBundle\Form\Type\ChannelType</parameter>
         <parameter key="sylius.context.channel.class">Sylius\Component\Channel\Context\ChannelContext</parameter>
     </parameters>
 
     <services>
+        <service id="sylius.form.type.channel" class="%sylius.form.type.channel.class%">
```

Comment:
This and the parameter is already handled by https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml#L46